bin/rcup.in

```diff
@@ -126,8 +126,20 @@ is_nested() {
   echo "$1" | sed "s:$DEST_DIR/::" | grep -q '/'
 }
 
+is_linked() {
+  local src="$1"
+  local dest="$2"
+
+  if [ -h "$dest" ]; then
+    local link_dest=$(readlink $dest)
+    [ "$link_dest" == "$src" ]
+  else
+    return 1
+  fi  
```

Comment:
Beware that `-a` doesn't short-curcuit

```sh
% test 0 = 1 -a "$(echo lol >&2)" = x
lol
```

If that matters (I think `readlink` errors on non-links), this may be better:

```sh
[ -h "$dest" ] && [ "$(readlink "$dest")" = "$src" ]
```